src/plugins/console/server/lib/spec_definitions/json/generated/cat.allocation.json

```diff
@@ -5,7 +5,7 @@
       "bytes": [
         "b",
         "k",
-        "osd",
+        "kb",
```

Comment:
My guess is that these are regressions? Naively, I'd think `osd` is the correct value.

Reply:
This got replaced on running the spec generation script with latest OpenSearch "main" branch. Is there a way I can verify the correct value?

Reply:
`kb` is the right value. When we forked and did renaming we just went ahead and just renamed a lot of references. This `kb` is related to `bytes` so it we seem we had an existing bug.